1.0.2 doc changes

cs-abhishek-shukla opened the pull request:
- Fixed Typo, Grammar, label names, doc structure, and information flow issues

cs-rohit-khune commented on docs/contents.md:
```diff
@@ -23,8 +23,8 @@
-| Check for SLA violation<sup>New</sup>          | Checks for breach notification violation and update SLA status to Missed                                                |
-| Reminder for SLA Breach<sup>New</sup>          | Sends reminder for the notification SLA Breach                                                                          |
+| Check for SLA violation                        | Checks for breach notification violation and update SLA status to Missed                                                |
+| Reminder for SLA Breach                        | Sends reminder for the notification SLA Breach                                                                          |
```
This playbook is repeated in the table.  Refer to line no 37

cs-abhishek-shukla replied:
Fixed @cs-rohit-khune

cs-rohit-khune commented on docs/contents.md:
```diff
@@ -38,7 +38,12 @@
 | Send Data Breach Report to DPA                 | Send Data Breach Report to Data Protection Authority                                                                    |
 | Set Breach Notification SLA                    | Sets Breach Notification SLA                                                                                            |
```
`Set Breach Notification SLA` playbook name is repeated in a table

cs-abhishek-shukla replied:
Fixed @cs-rohit-khune

cs-rohit-khune commented on release_notes.md:
```diff
@@ -1,11 +1,23 @@
+<table>
+    <tr>
+        <td><strong>NOTE</strong></td>
+        <td>This solution pack is now uncertified.</td>
```
@cs-abhishek-shukla Please remove this. The solution pack is now going certified

cs-abhishek-shukla replied:
Fixed @rohit

cs-rohit-khune reviewed (changes requested):
Added review comments

cs-rohit-khune commented on release_notes.md:
```diff
@@ -1,11 +1,23 @@
+
+### Data Compliance Module
+
 Newly Added Fields - Specifically for HIPAA Framework Solution pack
```
@cs-abhishek-shukla 
Please remove the below fields from the list

- Is A Breach Reportable
- What Health Information Was Exposed
- Reminder Mail Status

cs-abhishek-shukla replied:
Fixed @cs-rohit-khune

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -62,7 +62,89 @@ If you have already installed the GDPR solution pack v1.0.0, remove the followin
+
+1. Go to the *Create Data Compliance Record* playbook under the **10 - SP - GDPR Framework** collection and open the *Configuration* step
+
+    ![Execution Mode Configuration](./res/execution-mode-configuration.png)
```
Screenshot shows HIPPA on PB collection name

cs-abhishek-shukla replied:
Fixed @cs-dev-chaudhary

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -62,7 +62,89 @@ If you have already installed the GDPR solution pack v1.0.0, remove the followin
+        </tr>
+    </table>
+
+    ![Execution Mode Configuration Parameters](./res/execution-mode-configuration-parametes.png)
```
Same as above.

cs-abhishek-shukla replied:
Fixed @cs-dev-chaudhary

cs-abhishek-shukla committed:
1.0.0 doc changes
- Fixed Typo, Grammar, label names, doc structure, and information flow issues
- Moved Playbook Execution Modes to Configuration under Setup.md
- Moved fields and picklists to Contents
